lightwood/analysis/helpers/mavg.py

```diff
@@ -0,0 +1,66 @@
+from copy import deepcopy
+from typing import Optional, Tuple, Dict
+from types import SimpleNamespace
+
+import pandas as pd
+import numpy as np
+
+from lightwood.analysis.base import BaseAnalysisBlock
+from lightwood.helpers.log import log
+
+
+class MAvg(BaseAnalysisBlock):
+    """
+    Wrapper analysis block for the 'Robbie'...
+    """
+
+    def __init__(self, deps: Optional[Tuple] = ()):
+        super().__init__(deps=deps)
+
+    def analyze(self, info: Dict[str, object], **kwargs) -> Dict[str, object]:
```

Comment:
fun fact, you can now use `dict` here instead of `Dict` (same for `list` and `tuple` instead of `List` and `Tuple`) thus getting rid of the typing imports, most lang servers will support the new type hints

Reply:
I'm still unsure whether there's any benefit at all with using these `typing` aliases to the actual type or not. I did some reading on what you mention here and it appears to be supported starting 3.9, so I'm tempted to keep it like this for a bit longer until we move out of 3.7 and 3.8.